<launch>
    <rosparam command="load" file="$(find MVS)/config/camera.yaml" ns="camera" />
    <node pkg="MVS" type="MVS_node" name="MVS_node" respawn="true" output="screen" /> 
</launch>
